src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs

```diff
@@ -170,6 +172,20 @@ private void RestorePackages()
             }
         }
 
+        private object[] ProcessInputParameters(object[] parameters)
+        {
+            for (int i = 0; i < parameters.Length; i++)
+            {
+                TraceWriter writer = parameters[i] as TraceWriter;
```

Comment:
Note that the SDK also supports TextWriter parameters. TraceWriter is relatively new (added by me). I'm fine with saying that we only support log stream support for TraceWriter for now, but just a heads up if you see people try TextWriter. SDK users will try that.